libs/mini-rx-store-ng/src/lib/store.module.ts

```diff
@@ -22,7 +22,7 @@ export const FEATURE_CONFIGS = new InjectionToken<FeatureConfig<any>[]>(
 export function storeFactory(config: StoreConfig<Record<string, any>>, injector: Injector) {
     config.extensions =
         config.extensions && config.extensions.length
-            ? config.extensions.map((ext) => {
+            ? config.extensions.map((ext: any) => {
```

Comment:
normally the type of `ext` should be properly inferred by typescript.

I would suggest to remove any or explicitly add the StoreExtension type. 

![Screenshot 2024-04-30 at 20 02 03](https://github.com/spierala/mini-rx-store/assets/1272446/2b23e997-d570-4f6d-84ac-f2905cd840f0)